Projects/Presentation/Sources/MyPage/ViewModel/MypageViewModel.swift

```diff
@@ -67,6 +68,8 @@ final class MypageViewModel: ViewModel {
             if let url = URL(string: "https://complex-wombat-99f.notion.site/23ff4587491d80659ae3ea392afbc05e") {
                 externalURLPublisher.send(url)
             }
+        case .reportHistory:
+            break
```

Comment:
사실 지금 크게 노메러이긴 하지만 .resetGoal과 케이스 합쳐두 될 것 같아유
```suggestion
        case .resetGoal, .reportHistory:
            break
```

글구 `// 임시` url 주석도 이젠 없어두 될 것 같숨당 ~~